### DISCUSSION TITLE
Cover Card Icon Color

### DISCUSSION THREAD

**Discussion by snel6424** [piitaya/lovelace-mushroom#1122] [Ideas]
It would be great to be able to have the Cover Card have a color when the state is closed (green), similar to how the lock card icon is green when the entity is locked.

**Comment by SteveDockar** [comment]
Seconded! 

I use OpenGarage and would like to be able to show the state of the OG connection, particularly if it becomes unreachable for some reason.

**Reply by snel6424** [reply]
See my response below on how I got this done.

**Comment by titis14** [comment]
Hi,
I also want to change the color of the icons like on the entities with "icon_color" I would like green closed and red open. In addition with the default icons the icon is complicated to see. Thanks
![firefox_2023-11-01_20-42-53](https://github.com/piitaya/lovelace-mushroom/assets/26297241/573456aa-1e55-4c83-ad9c-8a3ac1672de7)

**Reply by snel6424** [reply]
I fixed this issue by using the Tile card instead of the Mushroom Card

```
type: tile
entity: cover.garage_door_main
vertical: false
show_entity_picture: false
features:
  - type: cover-open-close
card_mod:
  style: |
    .tile {
      {% if is_state(config.entity, 'closed') %}
        --tile-color: var(--green-color);
      {% else %}  
        --tile-color: var(--red-color);
      {% endif %}  
    }
```
    
   
<img width="406" alt="image" src="https://github.com/piitaya/lovelace-mushroom/assets/45545357/6d2b021d-9cd0-4041-b161-7084634577f6">

**Reply by titis14** [reply]
Great thank you very much it's perfect with your code.

**Reply by piitaya** [reply]
You can directly use [theme state colors](https://www.home-assistant.io/integrations/frontend/#state-color)
In a theme file to be applied everywhere or using card mod if you want it only for the tile card.

```yaml
--state-cover-garage-open-color: var(--red-color)
--state-cover-garage-closed-color: var(--green-color)
```
